packages/pangea-sdk/pangea/services/sanitize.py

```diff
@@ -95,6 +135,44 @@ def sanitize(
         uploaded_file_name: Optional[str] = None,
         sync_call: bool = True,
     ) -> PangeaResponse[SanitizeResult]:
+        """
+        Sanitize
+
+        Apply file sanitization actions according to specified rules. Beta API.
```

Comment:
According to yesterday talk, should we bold `Beta API` or move next to the "title" `Sanitize`?

Reply:
I'm gonna tackle all of them (Share + Sanitize) at once after this.